src/js/state/Pools/types.ts

```diff
@@ -0,0 +1,82 @@
+import {Ts} from "../../brim"
+
+export type PoolsState = {
+  // workspaceId
+  [key: string]: {
+    // poolId
+    [key: string]: Pool
+  }
+}
+
+export type PoolsAction =
+  | POOLS_SET
+  | POOLS_DETAIL
+  | POOLS_RENAME
+  | POOLS_INGEST_PROGRESS
+  | POOLS_INGEST_WARNING_APPEND
+  | POOLS_INGEST_WARNING_CLEAR
+  | POOLS_REMOVE
+  | POOLS_WORKSPACE_REMOVE
+export type Pool = {
+  name: string
+  id: string
+  min_time: Ts
+  max_time: Ts
+  ingest: PoolIngest
+  size: number
```

Comment:
As part of the migration we can also delete existing `parent_id`s and `storage_kind`